vmdb/app/models/miq_provision_microsoft.rb

```diff
@@ -0,0 +1,10 @@
+class MiqProvisionMicrosoft < MiqProvision
+  include_concern 'Cloning'
+  include_concern 'Configuration'
```

Comment:
I think this is breaking the tests.  I don't see this file that's trying to be included.